doc/faq/customization.rst

```diff
@@ -183,4 +183,30 @@ Some tokens allow a special customization.
 
 The paper token allows you to add CSS for styling the printed output and
 add additional headers and footers. Read more about it at the
-paper token :ref:`paper_token_customize`.
\ No newline at end of file
+paper token :ref:`paper_token_customize`.
+
+.. _customize_3rd_party_tokens:
+
+New token classes
+~~~~~~~~~~~~~~~~~
+
+You can write your own Python token class, that should look similar to
+the Python modules located in the directory *privacyidea/lib/tokens/*.
```

Comment:
it should also inherit from/implement `privacyidea.lib.tokenclass.TokenClass` (which functions need to be implemented?)

Reply:
So you mean we should add some more meat here like, 
* inheritance from which classes, 
* slight description of which method is probably necessary to have.

However, I did not want to write a complete development guide ;-) 
But I will add some.

Reply:
I do not expect anyone to write a new 3rd party tokenclass soon.
I basically wrote this for us, to remember how to configure a new tokenclass, if we need to implement a token class, which we do not want to add to the core privacyIDEA.
However, I added some more meat in b7e67443.